custom/enikshay/tasks.py

```diff
@@ -99,6 +98,7 @@ def run(self):
                             e
                         )
                     )
+            bulk_update_cases(self.domain, updates)
```

Comment:
@sravfeyn i think it would still be good to check for len(updates) > 0 here otherwise i think it will perform an unnecessary empty form submission